velox/common/base/test_utils/CMakeLists.txt

```diff
@@ -13,3 +13,10 @@
 # limitations under the License.
 
 add_library(vector_operation_test_utils INTERFACE)
+
+add_executable(velox_succinctprint_test SuccinctPrinterTest.cpp)
+
+add_test(velox_succinctprint_test velox_succinctprint_test)
```

Comment:
@majetideepak Deepak, while working on landing this change, I realized that the test needs to be moved. Usually tests go into "tests" subdirectory, but here we don't seem to follow that convention. Instead, tests are placed in velox/common/tests. It would be nice to fix that, but for now, would you move the new test to velox/common/tests?

Reply:
@mbasmanova done.